osidb/models/tracker.py

```diff
@@ -160,6 +160,18 @@ def save(self, *args, bz_api_key=None, jira_token=None, **kwargs):
         if raise_validation_error:
             self.validate(dry_run=kwargs.get("no_alerts", False))
 
+        # TODO the following condition can be removed when we fully leave Buzilla
+        #
+        # there may still exist open trackers in Bugzilla at the time when the
+        # project their supposed to belong to migrated to Jira already which
+        # breaks the integration and as it is not simple to address but only
+        # temporary we agreed on a workaround of skiping the tracker sync
+        if affect := self.affects.first():
+            if ps_module := PsModule.objects.filter(name=affect.ps_module).first():
+                # tracker type and BTS mismatch
+                if self.TYPE2BTS[self.type] != ps_module.bts_name:
+                    return
```

Comment:
Instead of returning, shouldn't we skip sync and continue with a regular save with `super().save(*args, **kwargs)`? I'm not sure if this could break the save chain and leave other models in an improper state

Reply:
It would mean that we would update the `updated_dt` and desync with Bugzilla. The aim is really cut to the OSIDB-Bugzilla way of sync and only get the updates from there - mostly the eventual closure.

Reply:
But do we even care if it updates `updated_dt` and desyncs with Bugzilla? Isn't that the issue to begin with, that they are out of sync anyway so we skip them?